libvast/src/system/profiler.cpp

```diff
@@ -20,6 +20,7 @@
 #include "vast/logger.hpp"
 #include "vast/time.hpp"
 
+#include <caf/event_based_actor.hpp>
```

Comment:
Unnecessary include.

Reply:
Actually, this is now necessary; see #928 from where I cherry-picked this commit for this branch to compile.